src/main/java/uk/gov/hmcts/reform/sscs/robotics/RoboticsJsonMapper.java

```diff
@@ -29,14 +31,17 @@
     private final DwpAddressLookupService dwpAddressLookupService;
     private final RegionalProcessingCenterService regionalProcessingCenterService;
     private final AirLookupService airLookupService;
+    private final boolean ucEnabled;
 
     @Autowired
     public RoboticsJsonMapper(DwpAddressLookupService dwpAddressLookupService,
                               RegionalProcessingCenterService regionalProcessingCenterService,
-                              AirLookupService airLookupService) {
+                              AirLookupService airLookupService,
+                              @Value("${feature.universal-credit.enabled}") boolean ucEnabled) {
```

Comment:
I guess this means we need a feature flag in any calling service now so at least Tribunals API?